src/main/java/gift/Model/Member.java

```diff
@@ -14,17 +14,25 @@ public class Member {
 
     protected Member() {}
 
-    public Member(Long id, String email, String password) {
-        this.id = id;
+    public Member(String email, String password) {
+        validateEmail(email);
+        validatePassword(password);
+
         this.email = email;
         this.password = password;
     }
 
-    public Member(String email, String password) {
-        this.email = email;
-        this.password = password;
+    public void validateEmail(String email){
+        if (email.isBlank() || email == null)
```

Comment:
email에 null이 오면 nullPointException이 발생할 수 있습니다.
null체크는 항상 맨 먼저 해주는게 좋습니다.
```suggestion
        if (email == null || email.isBlank())
```

Reply:
넵! 수정하겠습니다!